cpp/src/arrow/flight/sql/odbc/flight_sql/utils_test.cc

```diff
@@ -27,6 +28,13 @@
 namespace driver {
 namespace flight_sql {
 
+class UtilTestsWithCompute : public ::testing::Test {
+ public:
+  // This must be done before using the compute kernels in order to
+  // register them to the FunctionRegistry.
+  void SetUp() override { ASSERT_OK(arrow::compute::Initialize()); }
```

Comment:
`arrow::compute::Initialize()` is called for each test, right?

Can we use `::testing::Environment` to call `arrow::compute::Initialize()` only once?
See also:
* https://google.github.io/googletest/advanced.html#global-set-up-and-tear-down
* https://google.github.io/googletest/advanced.html#global-set-up-and-tear-down